src/Cache/_ScsDataClient.php

```diff
@@ -70,4 +74,19 @@ public function get(string $cacheName, string $key) : CacheGetResponse
         return new CacheGetResponse($response);
     }
 
+    public function delete(string $cacheName, string $key) : CacheDeleteResponse
+    {
+        validateCacheName($cacheName);
+        try {
+            $deleteRequest = new _DeleteRequest();
+            $deleteRequest->setCacheKey($key);
+            $call = $this->grpcManager->client->Delete($deleteRequest, ["cache" => [$cacheName]], ["timeout" => $this->deadline_seconds]);
+            [$response, $status] = $call->wait();
+        } catch (Exception $e) {
+            throw $e;
+        }
```

Comment:
whats going on here w/ try/catch and rethrow? can we just erase whole thing was this left over from testing something?

Reply:
This is set up for our future error handling where we don't bubble up exceptions. I've added comments to make that clearer.